index.js

```diff
@@ -45,7 +45,13 @@ const converter = createConverter({
 let timer = process.hrtime();
 
 const dotnetProject = path.join(__dirname, 'lib/csharp-models-to-json');
-const dotnetProcess = spawn('dotnet', ['run', `--project "${dotnetProject}"`, `"${path.resolve(configPath)}"`], { shell: true });
+
+let processParamaters = ['run', `--project "${dotnetProject}"`, `"${path.resolve(configPath)}"`];
+if(config.suppressEolFrameworkErrors){
+    processParamaters.push('--property:CheckEolTargetFramework=false');
+}
+
+const dotnetProcess = spawn('dotnet', processParameters, { shell: true });
```

Comment:
Hi @CharlieSaunders .
The variable `processParameters` is erronealy typed before as processParam**a**ters. Can you fix it?